catalog/core/catalog-core-commons/src/main/java/ddf/catalog/filter/delegate/ValidationQueryDelegate.java

```diff
@@ -57,17 +55,20 @@ public Boolean not(Boolean operand) {
 
     @Override
     public Boolean propertyIsEqualTo(String propertyName, String literal, boolean isCaseSensitive) {
-        return propertyName.equals(VALIDATION_ERRORS) || propertyName.equals(VALIDATION_WARNINGS);
+        return propertyName.equals(BasicTypes.VALIDATION_ERRORS)
```

Comment:
I think the comment at the top of the file answers this. This delegate checks to see if any of the filters are for either `VALIDATION_ERRORS` or `VALIDATION_WARNINGS`. Each of these methods gets called for a specific kind of filter (property is equal to, property is null, property is like), and will return true when the property is either `VALIDATION_ERRORS` or `VALIDATION_WARNINGS`